update DW target platform name in sql projects

kisantia opened the pull request:
@scoriani mentioned in another PR that the official name for DW is now "Azure Synapse Dedicated SQL Pool".

![image](https://user-images.githubusercontent.com/31145923/160671773-cb12c00c-ee39-4a43-a28c-3c72d0048b91.png)

scoriani commented:
I would probably remove "Microsoft" at the beginning for consistency reasons. Otherwise we should repeat that also for SQL Server options which are equally Microsoft products, isn't it? 😊

kisantia commented:
> I would probably remove "Microsoft" at the beginning for consistency reasons. Otherwise we should repeat that also for SQL Server options which are equally Microsoft products, isn't it? 😊

Makes sense :) I removed the "Microsoft" from the Azure SQL database and Synapse target platforms and updated the screenshot in the description
